gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,656 @@
+
+.. Make sure that the status of your gateway is *Connected*. 
++
+If the status is *Disconnected*, it is likely that the cluster configuration is incorrect or there is a network issue. Check <<troubleshooting>> for more information.
+//TODO: COVER BOTH TYPES OF ISSUES; MAKE SURE MOST COMMON ARE COVERED
+//      don't have network-related troubleshooting yet; 
+//      so have some config troubleshooting
+
+[[create-api]]
+== Step 5: Publish and Deploy a Simple API to Flex Gateway
+
+Create and deploy a simple API to your gateway in Anypoint API Manager. 
+
+. In Runtime Manager, click the *View APIs* menu option for your gateway to navigate to its *View APIs* page.
++
+image::gateway-quickstart-k8-view-apis.png["View APIs menu item for Flex Gateway"]
++
+Alternatively, you can navigate to API Manager from the Anypoint Platform menu.
+. From the page that opens, click *Add API* (and if present, *Add new API*) to navigate to the *APIs / Add API* page in API Manager.
+. From *APIs / Add API*, configure an API by working through the API configuration pages, starting with the *Runtime* configuration:
++
+image::gateway-quickstart-k8-select-gateway-for-api.png["Add API page with Flex Gateway Runtime and my-gateway selected"]
++
+.. From the *Runtime* configuration page, select *Flex Gateway* as your runtime.
+.. Find and select your gateway in the *Select a gateway* area of the page.
++
+Use the search field to locate it, if necessary.
+.. Click *Next* to open the *API* page.
+
+. In the *API* configuration page associated with *APIs / Add API*, provide an API name and asset type:
++
+image::gateway-quickstart-k8-api-config-api.png["Screenshot of 'API' page",width=60%]
++
+.. Click *Create new API*.
++
+The remaining procedures walk you through steps for creating and testing a new API that is running on your gateway. Notice, however, that it is also possible to select an existing API from Exchange from this page. That process is described in the xref:api-gateway::create-instance-task.adoc#add-a-new-api[Add a New API] within the API Manager documentation.
++
+.. Provide an API name, such as *my-api*.
+.. Select *HTTP API* as the asset type.
+.. Click *Next* to open the *Endpoint* configuration page.
+
+. In the *Endpoint* page associated with *APIs / Add API*, set up an endpoint for your API:
++
+image::gateway-quickstart-k8-api-config-endpoint.png["Screenshot of 'Endpoint' page",width=90%]
++
+.. Copy the following URI into the *Implementation URI* field:
++
+[source]
+----
+https://jsonplaceholder.typicode.com/
+----
+.. Type `80` in the *Port* field.
+.. Click *Next* to open the *Review* page.
+
+. In the *Review* page associated with *APIs / Add API*: 
+.. Review your API configuration.
+.. If necessary, use *Edit* to revise any configurations. 
+.. When your configuration is correct, click *Save & Deploy*.
++
+Inside the API instance, all HTTP requests to the external IP associated with your gateway are proxied to the `jsonplaceholder` service.
```

Comment:
ok, that language came from here: https://dev-docs-internal.kqa.msap.io/beta-flex-gateway-get-well/gateway/1.4/flex-gateway-getting-started#publish-a-simple-api 

Should that change, as well?

Reply:
Right, there it says `inside the gateway instance Docker container` instead of `API instance`. We can also say it like this:

```suggestion
Inside the gateway, all HTTP requests made to `localhost` on port 80 are proxied to the `jsonplaceholder` service.
```

But I would like to avoid mentioning `localhost` because it might be confusing (it's the localhost of the container running in the remote machine, not the localhost where the user is running the commands). Therefore I suggest `incoming HTTP requests`:

```suggestion
Inside the gateway, all incoming HTTP requests on port 80 are proxied to the `jsonplaceholder` service.
```

Reply:
Guess I need some coffee! 😆 I'll commit the last change.